com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -766,6 +768,7 @@ public void Dispose()
         internal SceneEventData(NetworkManager networkManager)
         {
             m_NetworkManager = networkManager;
+            SceneEventId = Guid.NewGuid();
```

Comment:
nit: instead of carrying `Guid` type around, it could be fancier to do a `.ToString()` on the `Guid` instance and then `XXHash` that `string` to convert it to an `int` hash value.

Reply:
Hmmm... I like your brain.